docs/public-networks/how-to/monitor/metrics.md

```diff
@@ -80,42 +79,42 @@ To configure Prometheus and run with Besu:
               - localhost:9545
     ```
 
-</TabItem>
-
-</Tabs>
+    </TabItem>
+    </Tabs>
 
     Prometheus requires 3 MB of space per node per hour for metrics, with a `scrape_interval` of 15 seconds.
 
-2.  Start Besu with the [`--metrics-enabled`](../../reference/cli/options.md#metrics-enabled) option. To start a single node for testing with metrics enabled, run the following command:
-
-<Tabs>
+2.  Start Besu with the [`--metrics-enabled`](../../reference/cli/options.md#metrics-enabled) option.
+    To start a single node for testing with metrics enabled, run the following command:
 
-<TabItem value="Syntax" label="Syntax" default>
+    <Tabs>
+    <TabItem value="Syntax">
 
     ```bash
     besu --network=dev --miner-enabled --miner-coinbase <COINBASE ADDRESS> --rpc-http-cors-origins="all" --rpc-http-enabled --metrics-enabled
     ```
 
-</TabItem>
-
-<TabItem value="Example" label="Example">
+    </TabItem>
+    <TabItem value="Example">
 
     ```bash
     besu --network=dev --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-http-cors-origins="all" --rpc-http-enabled --metrics-enabled
     ```
 
-</TabItem>
-
-</Tabs>
+    </TabItem>
+    </Tabs>
 
-    To specify the host and port on which Prometheus accesses Besu, use the [`--metrics-host`](../../reference/cli/options.md#metrics-host) and [`--metrics-port`](../../reference/cli/options.md#metrics-port) options. The default host and port are 127.0.0.1 (`localhost`) and 9545.
+    To specify the host and port on which Prometheus accesses Besu, use the
+    [`--metrics-host`](../../reference/cli/options.md#metrics-host) and
+    [`--metrics-port`](../../reference/cli/options.md#metrics-port) options.
+    The default host and port are `127.0.0.1` (`localhost`) and `9545`.
```

Comment:
The default host and port are `127.0.0.1` (`localhost`) and `9545`. = The default host and port are `127.0.0.1` (`localhost`) and `9545`, respectively.  ?  This edit may not be necessary with the mention of "(`localhost`)".